packages/core/src/tests/runtimes/node.test.ts

```diff
@@ -0,0 +1,32 @@
+import execa from "execa";
+import { createFixture } from "fs-fixture";
+
+describe("Node: System", () => {
+  test("ESM", async () => {
+    const fixture = await createFixture("./src/tests/fixtures/esm");
+
+    await execa("npm", ["link", "llamaindex"], { cwd: fixture.path });
+
+    const result = await execa("node", [fixture.path]);
+
+    expect(result.stdout).toContain(
+      "The author attended college but did not find it particularly interesting or relevant to their programming interests. They slacked off, skipped lectures, and eventually stopped attending altogether. They returned their Macbook and only went back to the college five years later to pick up their papers.",
```

Comment:
So this kind of thing probably won't work with the LLM, because a certain percentage of the time it'll output something different, even if we do temp = 0.

Reply:
Nice, yeah thats what I figured. I will push up a version with exports and method testing for us to take a look at!